src/QMCWaveFunctions/lcao/MultiQuinticSpline1D.h

```diff
@@ -305,8 +336,11 @@ struct MultiQuinticSpline1D
       }
     }
   }
+  
+  int getNumSplines() const { return num_splines_; }
+  void setNumSplines(int num_splines) { num_splines_ = num_splines; }
```

Comment:
Add inline keyword before these two functions.

Reply:
inline is implicit, if you want it to avoid a multiple definition error, which I haven't see yet it could be added if warranted but I'd rather see that, that's a bad code smell that should be justified.

Reply:
Class member function should have no multi-definition issue anyway. inline is never enforced it is just a suggestion the the compiler. Since the function is so small, I expect the compiler to do the inlining anyway.

Reply:
You are right because class member function defined in the class declaration are implicitly inline so its always redundant. If it is advantageous to inline the compiler will. What should happen here is all the other member definitions need inline removed.

Reply:
I don't think all the member functions are inlined. If they are big, they will not be inlined.
The linker knows how to deal with multi definition of class member functions.
inline keyword is always a suggestion to the compiler. The existing use of inline is appropriate I think and there is no need of removal.

Reply:
"A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function. "